data-raw/process_data_sets_maggie.R

```diff
@@ -0,0 +1,266 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(readr)
+install.packages("janitor")
+library(janitor)
+
+# nba-carmelo---------------------------------------------------------------------
+nba_carmelo <- read_csv("data-raw/nba-carmelo/nba_elo.csv")
+colnames(nba_carmelo) <- colnames(nba_carmelo) %>%
+  tolower() %>%
+  str_replace_all(" ", "_")
+nba_carmelo <- nba_carmelo %>%
+  mutate(
+    team1 = as.factor(team1),
+    team2 = as.factor(team2),
+    playoff = ifelse(playoff == "t", TRUE, FALSE),
+    playoff = ifelse(is.na(playoff), FALSE, TRUE),
+    neutral = ifelse(neutral == 1, TRUE, FALSE))
+#TODO: Discuss: data could be narrower if separated elo_score and carmelo_score then also had type (1_pre, 2_pre, prob1, prob2, 1_post, 2_post) but I'm not sure if this is as intuitive
+devtools::use_data(nba_carmelo, overwrite = TRUE)
+
+# nfl-elo---------------------------------------------------------------------
+nfl_elo <- read_csv("data-raw/nfl-elo/nfl_elo.csv")
+colnames(nfl_elo) <- colnames(nfl_elo) %>%
+  tolower() %>%
+  str_replace_all(" ", "_")
+nfl_elo <- nfl_elo %>%
+  mutate(
+    team1 = as.factor(team1),
+    team2 = as.factor(team2),
+    playoff = ifelse(playoff == "t", TRUE, FALSE),
+    playoff = ifelse(is.na(playoff), FALSE, TRUE),
+    neutral = ifelse(neutral == 1, TRUE, FALSE))
+devtools::use_data(nfl_elo, overwrite = TRUE)
+
+# nfl-fandom---------------------------------------------------------------------
+nfl_fandom_google <- read_csv("data-raw/nfl-fandom/NFL_fandom_data-google_trends.csv", skip=1)
+names(nfl_fandom_google) <- tolower(names(nfl_fandom_google))
+nfl_fandom_google <- nfl_fandom_google %>% 
+  rename(
+    trump_2016_vote = 'trump 2016 vote%'
+  ) %>%
+  mutate(
+    dma = as.factor(dma),
+    nfl = as.numeric(str_replace_all(nfl, "%", "")), 
+    nba = as.numeric(str_replace_all(nba, "%", "")), 
+    mlb = as.numeric(str_replace_all(mlb, "%", "")), 
+    nhl = as.numeric(str_replace_all(nhl, "%", "")), 
+    nascar = as.numeric(str_replace_all(nascar, "%", "")), 
+    cbb = as.numeric(str_replace_all(cbb, "%", "")), 
+    cfb = as.numeric(str_replace_all(cfb, "%", "")), 
+    trump_2016_vote = as.numeric(str_replace_all(trump_2016_vote, "%", ""))
+      )
+#TODO: should I create a sport variable to get rid of nfl, nba etc to make this more long/narrow? How would I do that?
+devtools::use_data(nfl_fandom_google, overwrite = TRUE)
+
+nfl_fandom_surveymonkey <-read_csv("data-raw/nfl-fandom/NFL_fandom_data-surveymonkey.csv", skip=1)
+#TODO: Clarify: unsure of why there are three variables for each race (with different values) for each team-- maybe stats for people who chose that team as first second and third favorite? 
```

Comment:
Minor ["style guide"](http://style.tidyverse.org/syntax.html#long-lines) nitpick: try to keep lines under 80 characters so that they are easy to read; you can quickly do this by highlighting the line in question and pressing COMMAND+SHIFT+/ on a macOS

As to your question, not sure. Was there anything in the article talking about this?